### ISSUE TITLE
Code review failing for reasons unknown

### ISSUE THREAD

**Issue by michaelcullum** [schmittjoh/php-code-analysis#35]
http://jmsyst.com/reviews/1483cb73-a6c7-4229-ab8d-0ef6c725d897

Not quite sure why it keeps failing.

**Comment by michaelcullum** [comment]
Same for http://jmsyst.com/reviews/091c531c-9740-4341-8155-22ebd97b6de1

**Comment by schmittjoh** [comment]
It seems like Swiftmailer fails when it tries to send you an email (there
is a space too much at the end).

I'll add a trim in the next deployment, but would be cool if you could
manually change it, and verify that it works then.

**Comment by michaelcullum** [comment]
At the end of what?

**Comment by schmittjoh** [comment]
The email address: `...@phpbb.com\s`

This could also be a `\r` if you have Mac line endings by any chance? Anyway, will deploy a fix for it soon.

**Comment by michaelcullum** [comment]
Not sure where it is trying to send an email? I haven't touched swiftmailer on it yet. And me and the other person working on it have UNIX line endings so not sure how that happened.

**Comment by schmittjoh** [comment]
No, it's not in your code. It's one of the notification emails that you have configured (one has a space, or another whitespace character at the end).

**Comment by michaelcullum** [comment]
Oh right. Thanks.

**Comment by michaelcullum** [comment]
Is there a way to force an instant code review?

**Comment by stof** [comment]
@unknownbliss There is a button to schedule a full review

**Comment by michaelcullum** [comment]
Not quite sure how I missed that. Thanks.

**Comment by michaelcullum** [comment]
Works, thanks.
